res/lang/programme/programme_en.txt

```diff
@@ -106,46 +106,46 @@ PROGRAMME_PRODUCT_SHOW        = Show
 PROGRAMME_PRODUCT_FEATURE     = Feature
 PROGRAMME_PRODUCT_INFOMERCIAL = Infomercial
 PROGRAMME_PRODUCT_EVENT       = Event
-PROGRAMME_PRODUCT_MISC        = Misc
+PROGRAMME_PRODUCT_MISC        = Misc.
 
 PROGRAMME_IN_PROGRAMME_PLAN              = Programme in programme plan!
 TRAILER_IN_PROGRAMME_PLAN                = Trailer in programme plan!
 PROGRAMME_AND_TRAILER_IN_PROGRAMME_PLAN  = Programme and trailer in programme plan!
 ONLY_1_BROADCAST_POSSIBLE                = Only 1 broadcast possible!
 ONLY_X_BROADCASTS_POSSIBLE               = Only %X% broadcasts possible!
 NO_MORE_BROADCASTS_ALLOWED               = No more broadcasts allowed!
-BROADCAST_TIME_RESTRICTED                = Time of broadcast restricted
+BROADCAST_TIME_RESTRICTED                = Time of broadcast restricted!
 BROADCAST_ONLY_ALLOWED_FROM_X_TO_Y       = Broadcast only allowed from %X%:00 to %Y%:00!
 FIRST_BROADCAST_ONLY_ALLOWED_FROM_X_TO_Y = First broadcast only allowed from %X%:00 to %Y%:00!
 LIVE_PRODUCTION_FINISH_WILL_COST_X       = Live production finish will cost %X%
 
 OPENINGSHOW_TITLE       = Big launch of "%CHANNELNAME%"
-OPENINGSHOW_DESCRIPTION = Let's pop the corks. %CHANNELNAME% is launching today.
+OPENINGSHOW_DESCRIPTION = Let's pop the corks! %CHANNELNAME% is launching today.
 
 SPORT_PROGRAMME_TITLE                                = %SPORTNAME%: %LEAGUENAME% %SEASONYEARSTART%
 SPORT_PROGRAMME_ALL_X_MATCHES_OF_LEAGUEX_IN_SEASON_X = All %MATCHCOUNT% matches of %LEAGUENAME% in season %SEASONYEARSTART%.
 SPORT_PROGRAMME_MATCH_TIMES                          = Match times
-SPORT_PROGRAMME_MATCH_DESCRIPTION1                   = Match between place %TEAM1RANK% and %TEAM2RANK% of the league. %TEAM1NAMEINITIALS% coach %TEAM1TRAINERSHORT% is optimistic.
-SPORT_PROGRAMME_MATCH_DESCRIPTION2                   = %TEAM1NAMEINITIALS% star %TEAM1STARSHORT% against the team around top scorer %TEAM2STAR%.
+SPORT_PROGRAMME_MATCH_DESCRIPTION1                   = Match between teams placed %TEAM1RANK% and %TEAM2RANK% in the league. %TEAM1NAMEINITIALS% coach %TEAM1TRAINERSHORT% is optimistic.
+SPORT_PROGRAMME_MATCH_DESCRIPTION2                   = %TEAM1NAMEINITIALS% star, %TEAM1STARSHORT%, against top scorer %TEAM2STAR%'s team.
```

Comment:
That does not work for names ending with an s (eg. Thomas). Hence the original phrase was written with that in mind :)

Reply:
It does really, "Thomas's" is fine.  You can omit the s but you don't have to.